Makefile

```diff
@@ -18,23 +14,18 @@ endif
 
 build: $(BINARY)
 
-run: vendor/stamp
-	gom run $(BUILDFILES)
+run:
+	godep go run $(BUILDFILES)
```

Comment:
It shouldn't be necessary to run everything through `godep` when dependencies are vendored. The Go tool will find them automatically. `godep` should only be a requirement when you need to change/update the dependencies themselves. This is the really good thing about the new vendoring support.